packages/commonwealth/client/scripts/state/api/trpc/subscription/useCommunityAlertsQuery.ts

```diff
@@ -0,0 +1,5 @@
+import { trpc } from 'utils/trpcClient';
+
+export function useCommunityAlertsQuery() {
+  return trpc.subscription.getCommunityAlerts.useQuery({});
```

Comment:
we should pass staleTime param to the query, otherwise, it will be called way to often. Find `staleTime:` in the code and follow the pattern